src/scripts/main.js

```diff
@@ -1,29 +1,52 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  notification.classList.add('notification', type);
+
+  const titleElement = document.createElement('h2');
+  titleElement.classList.add('title');
+  titleElement.textContent = title;
+
+  const descriptionElement = document.createElement('p');
+  descriptionElement.innerHTML = description;
```

Comment:
Using `innerHTML` can lead to security risks if the `description` contains user-generated content. Consider using `textContent` instead to avoid potential XSS vulnerabilities.